rustls-symcrypt/README.md

```diff
@@ -48,6 +58,7 @@ Key exchanges are listed below, ordered by preference. IE: `SECP384R1` is prefer
 ```ignore
 SECP384R1
 SECP256R1
+SECP521R1
 X25519 // Enabled with the `x25519` feature

```

Comment:
To my understanding, many servers prefer x25519, however its not recommended by Microsoft SDL. 

By disabling by default we force hello retry request, when enabling `x25519` feature it is still last in prio and will still force a hello retry. Is this a speed trade off we are okay with?

Users can always set a custom config and set `x25519` as higher prio than `NistP256` or `NistP384`